pkg/api/types_swagger_doc.go

```diff
@@ -0,0 +1,956 @@
+		"secretRef": "Optional: SecretRef is name of the authentication secret for RBDUser, default is empty.",
+		"readOnly":  "Optional: Defaults to false (read/write). ReadOnly here will force the ReadOnly setting in VolumeMounts.",
+	}
+}
+
+func (RangeAllocation) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":      "RangeAllocation is an opaque API object (not exposed to end users) that can be persisted to record the global allocation state of the cluster. The schema of Range and Data generic, in that Range should be a string representation of the inputs to a range (for instance, for IP allocation it might be a CIDR) and Data is an opaque blob understood by an allocator which is typically a binary range.  Consumers should use annotations to record additional information (schema version, data encoding hints). A range allocation should *ALWAYS* be recreatable at any time by observation of the cluster, thus the object is less strongly typed than most.",
+		"range": "A string representing a unique label for a range of resources, such as a CIDR \"10.0.0.0/8\" or port range \"10000-30000\". Range is not strongly schema'd here. The Range is expected to define a start and end unless there is an implicit end.",
+		"data":  "A byte array representing the serialized state of a range allocation. Additional clarifiers on the type or format of data should be represented with annotations. For IP allocations, this is represented as a bit array starting at the base IP of the CIDR in Range, with each bit representing a single allocated address (the fifth bit on CIDR 10.0.0.0/8 is 10.0.0.4).",
+	}
+}
+
+func (ReplicationController) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":       "ReplicationController represents the configuration of a replication controller.",
+		"spec":   "Spec defines the desired behavior of this replication controller.",
+		"status": "Status is the current status of this replication controller. This data may be out of date by some window of time.",
+	}
+}
+
+func (ReplicationControllerList) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"": "ReplicationControllerList is a collection of replication controllers.",
+	}
+}
+
+func (ReplicationControllerSpec) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "ReplicationControllerSpec is the specification of a replication controller. As the internal representation of a replication controller, it may have either a TemplateRef or a Template set.",
+		"replicas": "Replicas is the number of desired replicas.",
+		"selector": "Selector is a label query over pods that should match the Replicas count.",
+		"template": "Template is the object that describes the pod that will be created if insufficient replicas are detected. Internally, this takes precedence over a TemplateRef. Must be set before converting to a v1beta1 or v1beta2 API object.",
```

Comment:
Please remove "Must be set before converting to a v1beta1 or v1beta2 API object." Thanks!